tensorflow/python/keras/optimizer_v2/adagrad.py

```diff
@@ -141,7 +141,7 @@ def from_config(cls, config, custom_objects=None):
         An optimizer instance.
     """
     if 'initial_accumulator_value' not in config:
-      config['initial_accumulator_value'] = 0.
```

Comment:
This is a backwards-incompatible change. TF cannot have backwards-incompatible changes.

Reply:
why changing a value from 0.0 to 0.1 is a backwards-incompatible change?